.changeset/five-onions-beg.md

```diff
@@ -0,0 +1,5 @@
+---
+'@finos/legend-graph': patch
+'@finos/legend-studio': patch
+'@finos/legend-studio-app': patch
+---
```

Comment:
Please add a changeset log. https://github.com/finos/legend-studio/blob/master/CONTRIBUTING.md#changeset. I think for this we would need a changelog for `legend-graph` with a `minor` patch. The others can be blank.

Reply:
Please split the changeset to 2 files. One with no message for `@finos/legend-studio` and `@finos/legend-manual-tests` since we're not making any significant changes here.

One with the message that you already had for `@finos/legend-graph`.